src/IOSurface/IOSurface.cs

```diff
@@ -0,0 +1,90 @@
+// 
+// IOSurface.cs
+//
+// Copyright 2016 Microsoft 
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+#if XAMCORE_2_0
+using System;
+using System.Runtime.InteropServices;
+using XamCore.CoreFoundation;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.IOSurface {
+	public partial class IOSurface {
+
+		// kern_return_t
+		[iOS (10,0), Mac(10,12)]
```

Comment:
I did not run this on older devices, but the API descriptions state that these APIs are available on iOS 10.   It is not the first time that Apple makes these APIs available retroactively.